ansible/README.md

```diff
@@ -14,6 +14,8 @@ sudo pip install ansible==2.0.2.0
 
 #### Vagrant users
 Nothing to be done, Ansible is already installed during vagrant provisioning.
+You can skip setup and prereq steps as those have been done by vagrant for you.  
+You may jump directly to [Deploying Using CouchDB](#deploying-using-couchdb)
 
 #### Mac users
 It is assumed that a VM has been provisioned using **docker-machine**.
```

Comment:
I went thru the process again on Mac starting from scratch and even using the tweak script and had problems with 4243, the default docker port 2376 works fine no need to force a different port